Frends.LDAP.SearchObjects/Frends.LDAP.SearchObjects/SearchObjects.cs

```diff
@@ -22,41 +22,65 @@ public class LDAP
     /// <returns>Object { bool Success, string Error, string CommonName, List&lt;SearchResult&gt; SearchResult }</returns>
     public static Result SearchObjects([PropertyTab] Input input, [PropertyTab] Connection connection, CancellationToken cancellationToken)
     {
-        if (string.IsNullOrWhiteSpace(connection.Host) || string.IsNullOrWhiteSpace(connection.User) || string.IsNullOrWhiteSpace(connection.Password))
-            throw new Exception("Connection parameters missing.");
+        if (string.IsNullOrWhiteSpace(connection.Host))
+            throw new Exception("Host is missing.");
```

Comment:
I think we lost username and pwd checks along the way, in case we are doing not anon.